release/BUILD

```diff
@@ -279,17 +279,38 @@ py_test(
 )
 
 py_test(
-    name = "air_benchmark_gpu_batch_prediction_smoke_test",
+    name = "air_benchmark_gpu_batch_prediction_parquet_smoke_test",
     size = "small",
     srcs = test_srcs,
     args = [
-        "--data-size-gb=1",
+        "--data-directory=10G-image-data-synethic-raw-parquet",
```

Comment:
I tried running the script locally. but it says "FileNotFoundError: air-example-data-2/10G-image-data-synethic-raw". 
Is this directory incorrect, or am I missing something?

Reply:
fixed